src/store/ChartStore.js

```diff
@@ -274,6 +283,53 @@ class ChartStore {
         stxx.append('createDataSet', this.updateComparisons);
     }
 
+
+    /**
+     * Get tradeTimes if not loaded yet
+     * OR update the active symbols by comapring open time
+     */
+    @action.bound updateMarketClosedStatus(api) {
+        const todayUtc = (new Date()).getTime();
+        const timeToEpochGMT = (hour) => {
```

Comment:
I double check your implementation, as we try to compare with GMT, this way get the GMT time

Reply:
They both get the GMT time. Note the `Z` at the back. Check the outputs:

```js
const timeToEpochGMTBruce = (time_stirng) => {
            const dateStr = new Date().toISOString().substring(0, 11);
            return new Date(`${dateStr}${time_stirng}Z`).getTime();
};

const timeToEpochGMT = (hour) => {
   let today = new Date(),
                time = hour.split(':');
            return Date.UTC(today.getUTCFullYear(), today.getUTCMonth(), today.getUTCDate(), time[0], time[1], time[2]);
         };
const time = "03:45:00";
console.log(timeToEpochGMTBruce(time));
console.log(timeToEpochGMT(time));
```